__docs__/wonder-blocks-popover/popover.stories.tsx

```diff
@@ -558,3 +558,21 @@ export const PopoverAlignment: StoryComponentType = {
         </View>
     ),
 };
+
+/**
+ * With custom aria-label - overrides the default aria-describedby and aria-labelledby
+ */
+
+export const WithCustomAriaLabel: StoryComponentType = {
+    args: {
+        children: <Button>Open popover</Button>,
+        content: ContentMappings.withTextOnly,
+        placement: "top",
+        dismissEnabled: true,
+        id: "",
+        initialFocusId: "",
+        testId: "",
+        onClose: () => {},
+        "aria-label": "",
```

Comment:
For the storybook case can you put in an aria-label just to highlight the experience? Thanks!

Reply:
+1 to this suggestion :)